controllers/notificationsController.js

```diff
@@ -0,0 +1,137 @@
+import logger from '../helpers/logger';
+import models from '../models/index';
+import Mail from '../helpers/sendMail';
+import Response from '../helpers/responseHelper';
+import { STATUS } from '../helpers/constants';
+/**
+ * @description This class represents a notification in the app
+ * @class NotificationsController
+ */
+class NotificationsController {
+  /**
+   * This function gets a user setting
+   * @param {Number} id - The message to be sent
+   * @returns {Promise} - returns a promise
+   */
+  static async getSetting(id) {
+    const userSetting = await models.Setting.findOne({ where: { userId: id } });
+    return userSetting;
+  }
+
+  /**
+   * This function gets a user setting
```

Comment:
#### Nitpick
- Is this a case of copy and paste?

Reply:
Yes, Sorry about that. I will resolve it now. I duplicated a function somewhere there.